src/invocation/ClusterService.ts

```diff
@@ -147,47 +174,34 @@ export class ClusterService implements Cluster {
             });
     }
 
-    clearMemberListVersion(): void {
-        this.logger.trace('ClusterService', 'Resetting the member list version.');
-        // This check is necessary so that when handling auth response, it will not
-        // intervene with client failover logic
-        if (this.memberListSnapshot !== EMPTY_SNAPSHOT) {
-            this.memberListSnapshot.version = 0;
-        }
-    }
-
-    clearMemberList(connectionRegistry: ConnectionRegistry): void {
-        this.logger.trace('ClusterService', 'Resetting the member list.');
-        // This check is necessary so that when handling auth response, it will not
-        // intervene with client failover logic
-        if (this.memberListSnapshot !== EMPTY_SNAPSHOT) {
-            const previousMembers = this.memberListSnapshot.memberList;
-            this.memberListSnapshot = new MemberListSnapshot(0, new Map<string, MemberImpl>(), []);
-            const events = this.detectMembershipEvents(previousMembers, [], connectionRegistry);
-            this.fireEvents(events);
-        }
+    // for test usage
+    getMemberListVersion(): number {
+        return this.memberListSnapshot.version;
     }
 
     handleMembersViewEvent(
         connectionRegistry: ConnectionRegistry,
+        clusterUuid: UUID,
         memberListVersion: number,
         memberInfos: MemberInfo[]
     ): void {
-        if (this.memberListSnapshot === EMPTY_SNAPSHOT) {
-            this.applyInitialState(memberListVersion, memberInfos)
-                .then(this.initialListFetched.resolve)
-                .catch((err) => {
-                    this.logger.warn('ClusterService', 'Could not apply initial member list.', err);
-                });
+        if (this.memberListSnapshot.version == INITIAL_MEMBER_LIST_VERSION) {
```

Comment:
Can you assign `this.memberListSnapshot` to a variable at the start of this function and use it like Java?

Reply:
Fixed.